### DISCUSSION TITLE
Please help can I import a video with transparent overlay

### DISCUSSION THREAD

**Discussion by MustardMonday** [keiwando/toonsquid#657] [Q&A] [answered]
I made a custom text for something I was working on and counted on being able to edit the video so remove the background but in order to do that I have to rasterise it which turns it into a single frame. How can I go about editing it so I can put it on top but still keep the video as is

Will I have to use an external editor to remove the background and if so what

I’ll include a quick snippet of the video I want to overlay if it helps for context

https://github.com/user-attachments/assets/857ab370-0cc5-4418-a648-6805df028b01

**Comment by budaloops** [comment]
Here’s one way you could do this

1. Take a screenshot of the dialogue box. Type in ‘remove BG’ on Google and use one of the free tools that pop up to remove the background. 

2. Then place the image of the dialogue box with transparent background into your scene.

3. Place a solid orange/ brown rectangle over the text to hide it. 

4. Find a similar font on dafont.com by searching for “pixel”. Here’s one: https://www.dafont.com/dogica.font?fpp=100&psize=s

5. Import the font into ToonSquid, then type in the text in the dialogue box and use the ‘End Trim’ property to animate it.

Also, the next major update should come with a chroma key (green screen) effect to remove the BG: https://toonsquid.com/updates/ToonSquid-2-Beta/

**Reply by MustardMonday** [reply]
This probably would’ve worked better and I’d like to use this next time but since I’ve already got so many of the dialogue sequences done maybe I’ll wait for the chrome key or something

**Comment by budaloops** [comment]
You could also trace over the dialogue box to crate your own

https://github.com/user-attachments/assets/a5a28585-aaa8-4712-aed4-1a079c31ee9e

**Comment by FoxeyeStudios** [comment]
That depends on a few factors.

1. Did you make the text animation in ToonSquid?
2. When you exported it, did you make sure you disabled the background layer?
3. Which Codec did you use when exporting, HEVC or ProRes 4444?

There are lots of different reasons why you’re having trouble and different work arounds. If you could explain in a bit more detail, I could give you a more helpful explanation.

**Reply by MustardMonday** [reply]
Sure thing. The original image with the text boxes has no background but I used capcut for the text so I couldn’t keep it with no background like I mentioned in another reply I probably could’ve made it in toonsquid had I have known how but now that I’ve already got so much of it done I’d like to figure a way to export it properly. As for codec I honestly don’t know I’m not very sophisticated hence why the program of choice was capcut but honestly can’t find any other way to do it maybe next time I’ll simply make it in the same program sorry this probably doesn’t explain much but should it come to it I’ll probably make everything a layer above and have an obnoxious black bar at the bottom of the screen

**Reply by FoxeyeStudios** [reply]
Hmmm… Ok. So you exported it through a video editor that doesn’t support alpha channels. There’s a work around for that. I’m going to make a video to walk you through how you can get rid of the black background without needing to redo the whole thing.

**Reply by FoxeyeStudios** [reply]
Ok, I made the video, but it was too large to post it on here. So, I have it on YouTube for you to watch it and see.

https://youtu.be/GPO89ro7dFE?si=l47x87DEpIFQNWVc

**Reply by budaloops** [reply]
These are great! 📝🤓👍

**Reply by MustardMonday** [reply]
> Ok, I made the video, but it was too large to post it on here. So, I have it on YouTube for you to watch it and see.
> 
> https://youtu.be/GPO89ro7dFE?si=l47x87DEpIFQNWVc

Thank you! I’ll get back to you on if/how this works once I get around to testing it!

**Reply by MustardMonday** [reply, marked as answer]
> Ok, I made the video, but it was too large to post it on here. So, I have it on YouTube for you to watch it and see.
> 
> https://youtu.be/GPO89ro7dFE?si=l47x87DEpIFQNWVc

So like, treat it as if I’m an absolute moron, because I totally am. Which one or combination of these would be exposure? The only one that seemed to work was the graph and making it fully white but it didn’t look the exact same as yours

Uploading FullSizeRender.MOV…

**Reply by FoxeyeStudios** [reply]
Ok, so I have two things I want to say.

1. Watch out for that self-deprecating… Not only is it a negative mindset, but it’ll take a toll on your mental health. Just because you don’t know something or you’re not as experienced as other people in a particular subject doesn’t mean you’re stupid. You’re not a moron if you have to keep asking questions, asking questions is something all smart people do because they want to learn. And that’s why this community exists, so that we’re able to help each other out. So don’t go beating yourself up because something isn’t as easy for you to understand as it is for others, that just means you can always improve.

2. I was a little confused as to why you were having trouble, so I downloaded CapCut to see what you were struggling with and I can see why you’re having problems. I suppose exposure isn’t a commonly used feature when it comes to consumer grade editing. That’s my bad, I just assumed all video editors included Exposure adjustment. In that case, you’ll need to use something different.

-make sure the saturation is all the way down
-use the graph curve feature to make it white. Make sure to only affect the main color channel (white dot)

Let me know how it turns out.

**Reply by FoxeyeStudios** [reply]
![IMG_4799](https://github.com/user-attachments/assets/9b7a59e1-74c1-4096-921a-29cc1776efd8)

**Reply by MustardMonday** [reply]
> Ok, so I have two things I want to say.
> 
> 1. Watch out for that self-deprecating… Not only is it a negative mindset, but it’ll take a toll on your mental health. Just because you don’t know something or you’re not as experienced as other people in a particular subject doesn’t mean you’re stupid. You’re not a moron if you have to keep asking questions, asking questions is something all smart people do because they want to learn. And that’s why this community exists, so that we’re able to help each other out. So don’t go beating yourself up because something isn’t as easy for you to understand as it is for others, that just means you can always improve.
> 2. I was a little confused as to why you were having trouble, so I downloaded CapCut to see what you were struggling with and I can see why you’re having problems. I suppose exposure isn’t a commonly used feature when it comes to consumer grade editing. That’s my bad, I just assumed all video editors included Exposure adjustment. In that case, you’ll need to use something different.
> 
> -make sure the saturation is all the way down -use the graph curve feature to make it white. Make sure to only affect the main color channel (white dot)
> 
> Let me know how it turns out.

I figured it out! I used one of the things you suggested so thanks sm for the help!!! : D
